src/components/cards/MiniCourseCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {ParsedCourseOverview} from "../../types/Course";
+import {firstLectureOfSubcourse} from "../../utils/CourseUtil";
+import moment from "moment";
+import classes from "./MiniCourseCard.module.scss";
+import Icons from "../../assets/icons";
+import { useHistory } from "react-router-dom";
+
+export const MiniCourseCard = (course: ParsedCourseOverview) => {
+  const firstLecture = firstLectureOfSubcourse(course.subcourse);
+  const firstLectureDate = moment.unix(firstLecture.start).format("DD.MM.");
+
+  const history = useHistory();
+
+  return (
+    <div className={classes.courseCard} onClick={() => history.push(`/courses/${course.id}`)}>
+      <div className={classes.highlight} />
+      <div className={classes.courseTitle}>
+        { course.name }
+      </div>
+      <div className={classes.courseOutline}>
+        { course.outline }
+      </div>
+
+      <div className={classes.courseInformation}>
+        <div className={classes.courseDate}>
+          <Icons.Grade style={{width: "25px", height: "25px"}} />
```

Comment:
I think this should be `Icons.Calendar` instead of `Icons.Grade`

Reply:
Fixed. Thank you! :)